src/Agent.Worker/Agent.Worker.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="vss-api-netcore" Version="0.5.95-private" />
+    <PackageReference Include="vss-api-netcore" Version="0.5.97-private" />
```

Comment:
@TingluoHuang please review this package update.

Reply:
you should change reference to `vss-api-netcore` in all csprojs under agent repo.

Reply:
updated all csprojs